src/api/world.ts

```diff
@@ -258,6 +258,19 @@ export const updateWorldScheduleSettings = async (
   )(await createFirestoreWorldScheduleInput(world));
 };
 
+export const addWorldAdmin = async (worldId: WorldId, userId: UserId) =>
+  await httpsCallable(
+    FIREBASE.functions,
+    "world-addWorldAdmin"
```

Comment:
[NOTE] [OOS]
I think we can put these strings on a frozen object for reference and maintainability.
A single utility file with something like https://github.com/sparkletown/sparkle/pull/2914/files#diff-2e14c70c6dbf9de86687e6d3b6a66d3f8492cccd3bf6d450d5324ef2a7d2687eR28-R42